templates/compositions/ec-component-navigation/local-tasks.html.twig

```diff
@@ -0,0 +1,59 @@
+{#
+  Parameters:
+    - "title" (string) (default: 'Navigation'): for screen readers
+    - "links" (array) (default: []): collection of @ecl/link
+    - "variant" (string) (default: 'primary'): type of local tasks
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+#}
+
+{# Internal properties #}
+
+{% set _title = title|default('Navigation') %}
+{% set _links = links|default([]) %}
+{% set _variant = variant|default('primary') %}
+{% set _css_class = 'ecl-navigation-list' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if _variant == 'primary' %}
+  {% set _list_class = ' ecl-u-pb-m ecl-u-type-bold' %}
+  {% set _list_item_class = ' ecl-u-pa-m' %}
+{% endif %}
+{% if _variant == 'secondary' %}
+  {% set _list_class = ' ecl-u-pb-s ecl-u-type-s' %}
+  {% set _list_item_class = ' ecl-u-pa-s' %}
+{% endif %}
+{% if extra_classes is defined and extra_classes is not empty %}
```

Comment:
We don't need to "worry too much" if things are not set, defined etc. in Twig, the language parser should take care of that.

Let's just do ` {% if extra_classes is not empty %}`.